### PR TITLE
test cases for meta_adam

### PR DESCRIPTION
**Pull request by wangben718**

### CODE REVIEW

File: tests/testthat/test-independent-testing-meta_adam.R [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    observation = r2rtf::r2rtf_adae
+  )
+  expect_equal(class(test), "meta_adam")
+  expect_equal(test$plan, list())
+  expect_equal(test$population, list())
+  expect_equal(test$observation, list())
+  expect_equal(test$parameter, list())
+  expect_equal(test$analysis, list())
+  
+  meta <- meta_dummy()
```
Comment (LittleBeannie) [inline tests/testthat/test-independent-testing-meta_adam.R:16]: Do we need this (line 16 - 21)?

**Review by LittleBeannie — CHANGES_REQUESTED** [review]
Hi Ben, thanks for the test. I only have 1 minor comment.

File: tests/testthat/test-independent-testing-meta_adam.R (hunk 1 above)
Comment (wangben718) [reply tests/testthat/test-independent-testing-meta_adam.R:16]: > Do we need this (line 16 - 21)?

Yes, in order to test "The plan, observation, population, parameters, and analysis will be reported if one further define them into the metadata"

Comment (LittleBeannie) [reply tests/testthat/test-independent-testing-meta_adam.R:16]: But the `meta_adam` source code is just for initialization purposes. Plans and parameters are passed in later when you use `meta_build`. Please correct me if I misunderstand the code. I also updated the test plan of `meta_adam` and removed "The plan, observation, population, parameters, and analysis will be reported if one further defines them into the metadata".

**Review by LittleBeannie — APPROVED** [review]

**Pull request merged**